kura/org.eclipse.kura.nm/src/main/java/org/eclipse/kura/nm/NMDbusConnector.java

```diff
@@ -381,26 +385,32 @@ private void enableInterface(String deviceId, NetworkProperties properties, Devi
     private synchronized void manageNonConfiguredInterfaces(List<String> configuredInterfaces,
             List<Device> availableInterfaces) throws DBusException {
         for (Device device : availableInterfaces) {
-            NMDeviceType deviceType = getDeviceType(device);
-            String ipInterface = getDeviceId(device);
+            try {
+                NMDeviceType deviceType = getDeviceType(device);
+                String ipInterface = getDeviceId(device);
```

Comment:
Would it be a good idea to extract this into its own method so that we have symmetry with what the `manageConfiguredInterface` method does?

```suggestion
            try {
                manageNonConfiguredInterface(device);
```